docs/python/examples.ipynb

```diff
@@ -1686,7 +1686,418 @@
+    "    Node(1, Point(2.0, 0.0)),\n",
+    "    [\n",
+    "        basin.State(level=[2.5]),\n",
+    "        basin.Profile(area=[1000, 1000], level=[0.0, 3.0]),\n",
+    "        basin.Time(\n",
+    "            time=pd.date_range(model.starttime, model.endtime),\n",
+    "            drainage=0.0,\n",
+    "            potential_evaporation=0.0,\n",
+    "            infiltration=0.0,\n",
+    "            precipitation=0.0,\n",
+    "            urban_runoff=0.0,\n",
+    "        ),\n",
+    "    ],\n",
+    ")\n",
+    "model.basin.add(\n",
+    "    Node(4, Point(0.0, -2.0)),\n",
+    "    [basin.State(level=[1.5]), polder_profile, *basin_time],\n",
+    ")\n",
+    "model.basin.add(\n",
+    "    Node(6, Point(0.0, -4.0)),\n",
+    "    [basin.State(level=[1.0]), polder_profile, *basin_time],\n",
+    ")\n",
+    "model.basin.add(\n",
+    "    Node(8, Point(2.0, -4.0)),\n",
+    "    [basin.State(level=[1.5]), polder_profile, *basin_time3],\n",
+    ")\n",
+    "model.basin.add(\n",
+    "    Node(10, Point(4.0, -4.0)),\n",
+    "    [basin.State(level=[1.3]), polder_profile, *basin_time4],\n",
+    ")\n",
+    "model.basin.add(\n",
+    "    Node(12, Point(4.0, -2.0)),\n",
+    "    [basin.State(level=[0.1]), polder_profile, *basin_time],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After all the basins are set. The component inbetween the basins needs to be determined. For polder basin 5 (node 12), it needs to be maintain at level 0.0 meter. This means that either there should be no water in this basin, or the basin bottom is lower than sea level, and the water level should be maintained at sea level. \n",
+    "\n",
+    "If it is the first case, a pump is needed to make sure there is no water left in the basin. If it is the second case, a normal outlet with minimum crest level of 0.0 is needed. In this example, the first case is assumed. Therefore, a pump is set."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Setup the pumps:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model.pump.add(\n",
+    "    Node(13, Point(4.0, -1.0)),\n",
+    "    [pump.Static(flow_rate=[0.5 / 3600])],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "According to the description of situation 1 and 2, the water in one polder basin need to be able to flow to the downstream basin if the current basin has too much water that is higher than the setpoint or if the downstream basin requires more water. The functionality of the outlet with a minimum crest satisfy this feature. Take polder basin 1 (node 4) as an example. If the downstream polder has water shortage, polder basin 1 need to flow water as long as the water level is above 1.95 meter. So the minimum crest level for the outlet would be 1.95 meter."
```

Comment:
```suggestion
    "According to the description of situation 1 and 2, the water in one polder basin needs to be able to flow to the downstream basin if the current basin has too much water (i.e. the water level is above the setpoint) or if the downstream basin is below setpoint and needs more water. This could be modelled with an uncontrolled TabulatedRatingCurve node with Q=0 at the setpoint level (and Q rising when the level rises above setpoint) , or with an Outlet node where the minimum crest is specified at or just below the setpoint. In this example, we've chosen for the Outlet where we specify the minimum crest level 5 cm below the setpoint. For example: the Outlet of polder basin 1 (node 4) is specified with a minimum crest level of 1.95 meter."
```